javascript/ql/src/semmle/javascript/frameworks/SQL.qll

```diff
@@ -48,7 +54,7 @@ private module MySql {
   private class QueryCall extends DatabaseAccess, DataFlow::MethodCallNode {
     QueryCall() {
       exists(API::Node recv | recv = createPool() or recv = connection() |
-        this = recv.getMember("query").getReturn().getAUse()
+        this = recv.getMember("query").getASourceUse().getACall()
```

Comment:
It looks like `getASourceUse().getACall()` is quite a common pattern. Let's perhaps introduce a helper predicate that encapsulates it.

Reply:
Do you have any good ideas for naming? 
The best I can come up with is just `getACall()`.  

Unless we do something like `getADataFlowCall()`, but I think that is a bit too verbose.

Reply:
I think `getACall()` is not completely horrible.